src/qml/qmlapplication.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <coreservices.h>
+
+#include <QApplication>
+#include <QFileSystemWatcher>
+#include <QQmlApplicationEngine>
+
+namespace mixxx {
+namespace qml {
+
+class QmlApplication : public QObject {
+    Q_OBJECT
+  public:
+    QmlApplication(
+            QApplication* app,
+            std::shared_ptr<CoreServices> pCoreServices);
+    ~QmlApplication() = default;
+
+  public slots:
+    void loadQml(const QString& path);
+
+  private:
+    std::shared_ptr<CoreServices> m_pCoreServices;
+
+    QString m_mainQmlFilePath;
```

Comment:
`m_rootQmlFilePath`

Reply:
The QML file is named `main.qml` so I think this variable name is appropriate.